### PR TITLE
Add playwright emulation docs

### PR DESCRIPTION
**Pull request by tianfeng92**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/web-apps/automated-testing/_partials/_advanced-playwright.md [hunk 1]
```diff
@@ -36,3 +36,11 @@ const config: PlaywrightTestConfig = {
 :::caution
 A limitation on Sauce Labs exists with playwright 1.27.1 or below, where the user defined reporter is only honored when defined as an array: `reporter: [['./fancy.reporter.ts']]`. Using a string like `reporter: './fancy.reporter.ts'` would therefore always be ignored.
 :::
+
+## Emulation
+
+Playwright can test your app on any browser as well as emulate a real device such as a mobile phone or tablet. Check more details in [Playwright docs](https://playwright.dev/docs/emulation).
+
+:::note
+It is not a real Emulator or Simulator. Playwright just simulates the browser behavior such as `UseAgent`, `screenSize`, `viewport` and etc.
```
Comment (FriggaHel) [inline docs/web-apps/automated-testing/_partials/_advanced-playwright.md:45]: I guess this is `UserAgent` ?

```suggestion
It is not a real Emulator or Simulator. Playwright just simulates the browser behavior such as `userAgent`, `screenSize`, `viewport` and etc.
```

**Review by FriggaHel — APPROVED** [review]
LGTM, need probably a second look from a someone with a better english than mine :-)

File: docs/web-apps/automated-testing/_partials/_advanced-playwright.md [hunk 2]
```diff
@@ -36,3 +36,11 @@ const config: PlaywrightTestConfig = {
 :::caution
 A limitation on Sauce Labs exists with playwright 1.27.1 or below, where the user defined reporter is only honored when defined as an array: `reporter: [['./fancy.reporter.ts']]`. Using a string like `reporter: './fancy.reporter.ts'` would therefore always be ignored.
 :::
+
+## Emulation
+
+Playwright can test your app on any browser as well as emulate a real device such as a mobile phone or tablet. Check more details in [Playwright docs](https://playwright.dev/docs/emulation).
+
+:::note
+It is not a real Emulator or Simulator. Playwright just simulates the browser behavior such as `userAgent`, `screenSize`, `viewport` and etc.
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/_partials/_advanced-playwright.md:45]:
```suggestion
Playwright is not a real Emulator or Simulator. It just emulates the browser behavior such as `userAgent`, `screenSize`, `viewport` and etc.
```

File: docs/web-apps/automated-testing/_partials/_advanced-playwright.md [hunk 3]
```diff
@@ -36,3 +36,11 @@ const config: PlaywrightTestConfig = {
 :::caution
 A limitation on Sauce Labs exists with playwright 1.27.1 or below, where the user defined reporter is only honored when defined as an array: `reporter: [['./fancy.reporter.ts']]`. Using a string like `reporter: './fancy.reporter.ts'` would therefore always be ignored.
 :::
+
+## Emulation
+
+Playwright can test your app on any browser as well as emulate a real device such as a mobile phone or tablet. Check more details in [Playwright docs](https://playwright.dev/docs/emulation).
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/_partials/_advanced-playwright.md:42]:
```suggestion
With Playwright you can test your app on any browser as well as emulate a real device such as a mobile phone or tablet. See the [Playwright docs](https://playwright.dev/docs/emulation) for more details.
```

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**